README.md

```diff
@@ -7,7 +7,7 @@ An echo bot in multiple languages to get you started.
 | Language                                                     | core version |
 | ------------------------------------------------------------ | ------------ |
 | [C](./c)                                                     | `1.101.0`    |
-| [Go](./go)                                                   | `1.110.0`    |
+| [Go](./go)                                                   | `1.127.0`    |
```

Comment:
1.127.0 is not the core version but the client lib version, currently there is not a locking of core version for the lib, so it can be updated independently in theory unless it has a breaking change like changes in the JSONRPC API or new events

Reply:
did not want to track that, I think the value should represent what surely works, not what might still work.